components/benefit_sponsors/spec/models/benefit_sponsors/model_events/low_enrollment_notice_for_employer_spec.rb

```diff
@@ -53,6 +53,26 @@
         end
       end
     end
+
+    context "if benefit applicationis osse_eligible" do
+      subject { BenefitSponsors::Observers::NoticeObserver.new }
+      let(:model_event) { BenefitSponsors::ModelEvents::ModelEvent.new(:open_enrollment_end_reminder_and_low_enrollment, model_instance, {}) }
+
+      before do
+        params = {subject_gid: benefit_sponsorship.to_global_id, evidence_key: :osse_subsidy, evidence_value: 'true', effective_date: TimeKeeper.date_of_record }
+        result = ::Operations::Eligibilities::Osse::BuildEligibility.new.call(params)
+        eligibility = benefit_sponsorship.eligibilities.build(result.success.to_h)
+        eligibility.save!
+        year = model_instance.start_on.year
+        allow(EnrollRegistry).to receive(:feature?).with("aca_shop_osse_subsidy_#{year}").and_return(true)
+        allow(EnrollRegistry).to receive(:feature_enabled?).with("aca_shop_osse_subsidy_#{year}").and_return(true)
+      end
+
+      it "should trigger notice event" do
```

Comment:
should  `expect(subject.notifier).not_to receive(:notify)` be below the process_application_events call?

Reply:
Also should the statement be labeled "should NOT trigger notice event"?

Reply:
Alright great, in that case LGTM.  I